### DISCUSSION TITLE
Multi-Routing

### DISCUSSION THREAD

**Discussion by genrichards** [h2oai/wave#2258] [Q&A] [answered]
Routing:
In the current script, we utilize hash routing under @app("/") for routing e.g. #route1, #route2, #route3. 

However, the platform engineering team is interested in converting these into multiple routes, e.g @app("/luis"), @app("/claire"), @app("/sales"), etc. 

Is this possible within one app or do we need to deploy 3 separate apps? 

CC @kelvinbksoh @shivam5992

**Comment by mturoci** [comment, marked as answer]
3 separate apps would be needed.
